### ISSUE TITLE
Plowing crops or windrow gains no N or PK.

### ISSUE THREAD

**Issue by sootysax** [DeckerMMIV/FarmSim_Mod_SoilMod#15]
I'm trying to plow wheat, barley, alfala or straw windrow into the soil, but am seeing no change in the N or PK  levels.   Using a cultivator shows a small boost in N as expected.  

I thought this might be just on my new map, but I have tested this on several maps, and plowing these things results in no gains.   Plowing manure however works as expected.

on new Old Ridge Farm:
https://gist.github.com/sootysax/d5fd7fbb0424cfc39b0b/download

![fsscreen_2015_03_17_21_10_37](https://cloud.githubusercontent.com/assets/6591048/6700840/78fee8a2-ccea-11e4-9647-72aacd5cbc4e.png)

![fsscreen_2015_03_17_21_10_17](https://cloud.githubusercontent.com/assets/6591048/6700841/7dd8ca46-ccea-11e4-8da1-8b0cc6bd8dd7.png)

on new Thistle Farm:
https://gist.github.com/sootysax/10d2f5e7950236ccb82d/download

![fsscreen_2015_03_17_20_48_35](https://cloud.githubusercontent.com/assets/6591048/6700653/5c186de2-cce7-11e4-99ef-0fe4c02e3224.png)
![fsscreen_2015_03_17_20_48_06](https://cloud.githubusercontent.com/assets/6591048/6700660/64a64b00-cce7-11e4-8fb5-21604910244b.png)

**Comment by DeckerMMIV** [comment]
There was a discrepancy in priority for plugin:'Update foliage-layer for SoilMod' when looking at the `update-cultivator-area(before)` and `update-plough-area(before)`.

This should be fixed with 2.0.15-BETA, so please test if it fixes it.

Before:

```
SoilMod: Plugin for update-cultivator-area(before): (30) Destroy common area
SoilMod: Plugin for update-cultivator-area(before): (20) Update foliage-layer for SoilMod

SoilMod: Plugin for update-plough-area(before): (30) Destroy common area
SoilMod: Plugin for update-plough-area(before): (40) Update foliage-layer for SoilMod
```

The fix in 2.0.15-BETA:

```
SoilMod: Plugin for update-cultivator-area(before): (20) Update foliage-layer for SoilMod
SoilMod: Plugin for update-cultivator-area(before): (30) Destroy common area

SoilMod: Plugin for update-plough-area(before): (20) Update foliage-layer for SoilMod
SoilMod: Plugin for update-plough-area(before): (30) Destroy common area
```

**Comment by sootysax** [comment]
quick test seems to have fixed the problem...  i'll do some more testing and report any issue if they come up
